scripts/encrypt-secrets.sh

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 
 echo "Enter password for encryption: "
-read password
+read -s -p password
```

Comment:
Try:

    read -s -p "Enter password for encryption: " password
    echo

:-) The `-p` sets the prompt. (The `echo` afterwards is 'cuz apparently suppressing echo also suppresses the trailing endline...)